pkg/util/proxyutil/reverse_proxy.go

```diff
@@ -102,14 +102,16 @@ func errorHandler(logger glog.Logger) func(http.ResponseWriter, *http.Request, e
 			return
 		}
 
+		ctxLogger := logger.FromContext(r.Context())
```

Comment:
Should line `100` not also use the `ctxLogger`?

Reply:
Good catch :+1: I'll fix